docs/index-type-operator.md

```diff
@@ -63,7 +63,7 @@ type age = index<Person, key> -- Error message: Second argument to index<Person,
 ```
 Note: these errors will be part of the general type family reduction errors since `index` will be built into the type family system.
```

Comment:
```suggestion
Note: these errors will be part of the general type function reduction errors since `index` will be built into the type function system.
```